### ISSUE TITLE
Write definitions for glossaries that don't have them

### ISSUE THREAD

**Issue by mcharg** [ONRR/nrrd#1332]
Terms in this spreadsheet: https://teams.microsoft.com/l/file/73985E53-3C81-40A9-8853-A10482F267AB?tenantId=0693b5ba-4b18-4d7b-9341-f32f400a5494&fileType=xlsx&objectUrl=https%3A%2F%2Fdoimspp.sharepoint.com%2Fsites%2FNRRD%2FShared%20Documents%2FGeneral%2FNRRD%2FDesign%2FGlossaries%2Fglossary.terms.query.tool.xlsx&baseUrl=https%3A%2F%2Fdoimspp.sharepoint.com%2Fsites%2FNRRD&serviceName=teams&threadId=19:34f9f9ee6b0441dbacab56bfd552f819@thread.skype&groupId=3dd7454f-a781-414f-b6f1-b6d10fadb64a

**Comment by mcharg** [comment]
And make sure these ones are in the markdown file.

I think its missing terms for the units of measurement in
revenue: other revenue, not tied to commodity, NGL
production: mcf, bbl, ton and kilowatt hours, metric ton, long ton
disbursement: historic preservation fund, GOMESA, state 8(g)

and calender year and fiscal year

**Comment by mcharg** [comment]
Ticket/pounds definition is in #569

**Comment by Maroyafaied** [comment]
Ask Nadia for help on this.

**Comment by Maroyafaied** [comment]
@mcharg @jnwang01 I wrote definitions for all the missing fields. There are 3 terms that I'm waiting on the DORC team to provide definitions for. The only one I didn't know where it came from or what it stands for was "cyd"?

**Comment by mcharg** [comment]
@Maroyafaied Could cyd be cubic yards of debris?
![image](https://user-images.githubusercontent.com/38465926/113598507-3997fa00-960b-11eb-8c4d-d5a6ddf47625.png)

**Comment by mcharg** [comment]
![image](https://user-images.githubusercontent.com/38465926/113598753-98f60a00-960b-11eb-994d-f6dcace9d2fc.png)

**Comment by mcharg** [comment]
@Maroyafaied I highlighted a few in orange that I had comments on previously.  We need to edit other revenue now that it no longer includes civil penalties or inspection fees.  We wanted to add something to geothermal about why there are different units.  I added an explanation to the disbursements to [state name] about state being separate from county.

**Comment by Maroyafaied** [comment]
This is an explanation for Geothermal. 

Reporting geothermal resources relate first to the production’s usage—electrical generation or direct use—and second to its unit of measure. The unit of measure may be in terms of mass, such as pounds; volume, such as gallons; electricity, such as kilowatt hours (kWh); thermal energy, such as British thermal units (Btu); or other measurement units approved by ONRR. The unit of measure is usually either specified in the sales contract or dictated by the method used to value the production. 

- Geothermal byproducts: Geothermal byproducts are any mineral or minerals—other than oil, hydrocarbon gas, and helium—that are commercially recovered from produced geothermal fluids

Fluid geothermal resource: include steam, hot water, hot brines, and related heat and other associated energy . The two types of fluid geothermal resources in production usage are;      

Geothermal Direct Use: the utilization of geothermal for commercial, residential, agricultural, public facilities, or other energy needs other than the commercial production or generation of electricity. 

 Geothermal Electrical Generation: the utilization of geothermal resources to generate electricity for sale.

**Comment by Maroyafaied** [comment]
I will close this issue and make another issue to get these definitions from DORC. 

Other revenues
N/A
Oil & gas (pre-production)
Other Leasable Minerals
